spec/api.json

```diff
@@ -4564,15 +4371,15 @@
             },
             "description": "Ory Kratos is ready to accept connections."
           },
-          "500": {
+          "default": {
             "content": {
-              "application/json": {
+              "text/plain": {
```

Comment:
This looks phishy

Reply:
This change is related to https://github.com/ory/x/pull/685. The liveness check never returns a structured error. Any error the caller might see (if any) will be from cloudflare or the ingress controller.  Either way, it'll be unstructured stuff in their logs.

It was necessary to remove the structured error from the health/alive endpoint because of this change https://github.com/ory/herodot/pull/106, which was done months ago but never rolled out. The cleanest and simplest solution was just to get rid of the structured error alltogether.